plugins/utils/sql_utils.go

```diff
@@ -0,0 +1,58 @@
+package utils
```

Comment:
I would rename the package to be `sqlutil` (notice the singular as well).
wdyt? @GrayFlash @mabdh @ravisuhag 

so using it will be like e.g `sqlutil.FetchDBs`.
also reason for `sqlutil` instead of `sqlutils` is to follow go base library convention to use singular whenever possible.

Reply:
I know package name can be aliased when importing, but I think we can even do it from the package itself instead of aliasing it everywhere.

Reply:
Yes that would be more readable

Reply:
Yes @StewartJingga makes sense